tests/pyfunc/test_model_export_with_loader_module_and_data_path.py

```diff
@@ -93,6 +93,29 @@ def test_model_log_load(sklearn_knn_model, iris_data, tmpdir):
         sklearn_knn_model.predict(iris_data[0]), reloaded_model.predict(iris_data[0]))
 
 
+@pytest.mark.large
+def test_model_log_load_no_run(sklearn_knn_model, iris_data, tmpdir):
+    sk_model_path = os.path.join(str(tmpdir), "knn.pkl")
+    with open(sk_model_path, "wb") as f:
+        pickle.dump(sklearn_knn_model, f)
+
+    pyfunc_artifact_path = "pyfunc_model"
+    mlflow.pyfunc.log_model(artifact_path=pyfunc_artifact_path,
```

Comment:
As a sanity check, can we add a line that confirms `mlflow.active_run() == None` before we call `mlflow.pyfunc.log_model`?